docs/sources/operators-guide/running-production-environment-with-helm/configuration-with-helm.md

```diff
@@ -0,0 +1,282 @@
+
+## Example of configuration managed with Helm
+
+This example show how to set up the configuration to use an S3 bucket for blocks storage. We assume that the namespace in use is called `mimir-test`.
+
+1. Set up the external blocks storage, in this case S3 with buckets named for example `my-blocks-bucket`, `my-ruler-bucket` and in case of Grafana Enterprise Metrics `my-admin-bucket`.
+
+1. Create an external secret with the S3 credentials by writing the following to a `mysecret.yaml` file:
+
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: mimir-bucket-secret
+   data:
+     AWS_ACCESS_KEY_ID: FAKEACCESSKEY
+     AWS_SECRET_ACCESS_KEY: FAKESECRETKEY
+   ```
+
+   Replace FAKEACCESSKEY and FAKESECRETKEY with the actual value encoded with base64.
+
+1. Apply the secret to your cluster with the `kubectl` command:
+
+   ```bash
+   kubectl -n mimir-test apply -f `mysecret.yaml`
+   ```
+
+1. Prepare your custom values file called `custom.yaml`:
+
+   ```yaml
+   global:
+     extraEnvFrom:
+       - secretRef:
+         name: mimir-bucket-secret
+     podAnnotations:
+       bucketSecretVersion: "0"
+
+   # This turns of the built in MinIO support
+   minio:
+     enabled: false
+
+   mimir:
+     structuredConfig:
+       # Uncomment in case of Grafana Enterprise Metrics
+       # admin_client:
+       #   storage:
+       #     s3:
+       #       bucket_name: my-admin-bucket
+       #       access_key_id: ${AWS_ACCESS_KEY_ID}
+       #       endpoint: s3.amazonaws.com
+       #       secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+       alertmanager_storage:
+         s3:
+           bucket_name: my-ruler-bucket
+           access_key_id: ${AWS_ACCESS_KEY_ID}
+           endpoint: s3.amazonaws.com
+           secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+       blocks_storage:
+         backend: s3
+         s3:
+           bucket_name: my-blocks-bucket
+           access_key_id: ${AWS_ACCESS_KEY_ID}
+           endpoint: s3.amazonaws.com
+           secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+       ruler_storage:
+         s3:
+           bucket_name: my-ruler-bucket
+           access_key_id: ${AWS_ACCESS_KEY_ID}
+           endpoint: s3.amazonaws.com
+           secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+   ```
+
+1. Check the resulting configuration with the `helm` command before installing:
+
+   ```bash
+   helm -n mimir-test template mimir grafana/mimir-distributed -f custom.yaml -s templates/mimir-config.yaml
+   ---
+   # Source: mimir-distributed/templates/mimir-config.yaml
```

Comment:
[style advice from tech writers](https://github.com/grafana/technical-documentation/blob/main/docs/sources/style-guide/writing-conventions.md#command-line) is to separate the command and the output
```suggestion
   ```bash
   helm -n mimir-test template mimir grafana/mimir-distributed -f custom.yaml -s templates/mimir-config.yaml
    ```
   You should see the following output:
   ```yaml
   ---
   # Source: mimir-distributed/templates/mimir-config.yaml
```